### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by YuanLIN99825** [xyfJASON/ctrlora#21]
Thanks for the awesome work! 

My question is:

When fine-tuning ControlNet, is MSE used as the Loss Function?

**Comment by YuanLIN99825** [comment]
Thanks for the information!
